docs/dashboards/datagrid-component.md

```diff
@@ -0,0 +1,104 @@
+DataGrid Component
+===
+
+The DataGrid Component can be placed inside a data cell to allow the data visualization in the form of an editable data grid.
+
+<iframe style="width: 100%; height: 470px; border: none;" src="https://www.highcharts.com/samples/nonav/dashboards/datagrid-component/datagrid-options" allow="fullscreen"></iframe>
+
+
+## How to start
+
+1. To use the DataGrid Component in Dashboards, you need to import the appropriate modules.
+    ```html
+    <script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+    <script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+    <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+    ```
+
+    Alternatively, you can also use NPM packages (see: [Installation](https://www.highcharts.com/docs/dashboards/installation)) and import to connect them to the Dashboards.
+    ```ts
+    import * as Dashboards from '@highcharts/dashboards';
+    import DataGrid from '@highcharts/dashboards/es-modules/DataGrid/DataGrid';
+    import DataGridPlugin from '@highcharts/dashboards/es-modules/Dashboards/Plugins/DataGridPlugin';
+
+    DataGridPlugin.custom.connectDataGrid(DataGrid);
+    Dashboards.PluginHandler.addPlugin(DataGridPlugin);
+    ```
+
+2. You also need to import the styles.
+    ```css
+    @import url("https://code.highcharts.com/css/dashboards.css");
+    @import url("https://code.highcharts.com/css/datagrid.css");
+    ```
+
+3. After loading the necessary files, define a cell using a unique identifier for the cell, e.g. `dashboard-col-0`. This cell will be used to place the component in the dashboard.
+    ```js
+    gui: {
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                cells: [{
+                    id: 'dashboard-col-0'
+                }]
+            }]
+        }]
+    }
+    ```
+
+4. You will also need some data that you could display in your DataGrid component. For this purpose, you can, for example, define a connector that, in this case, downloads CSV data from the example URL.
```

Comment:
IMO, would be better to show user how to use in the simplest way and then add extra example with the connector.

(You can hardcode the CSV value).

Reply:
Scheduled after bug fix in the [dataTable](https://api.highcharts.com/dashboards/#interfaces/DataGrid_DataGridOptions.DataGridOptions-1#dataTable) option, which doesn't work for the DataGrid Component. At the moment changed from `csvURL` to `csv`.